guides/intro-to-ploomber/_source.md

```diff
@@ -0,0 +1,106 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.6
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+
+# Your first Python pipeline
+
+<!-- start description -->
+To run this example locally, [install Ploomber](https://docs.ploomber.io/en/latest/get-started/quick-start.html) and execute: `ploomber examples -n guides/first-pipeline`
+
+Found an issue? [Let us know.](https://github.com/ploomber/projects/issues/new?title=guides/first-pipeline%20issue) Have questions? [Ask us anything on Slack.](https://ploomber.io/community/)
+<!-- end description -->
+
+# Ploomber Tutorial Intro
+
+- This should showcase the Ploomber value proposition. 
+- We'll do a few ML operations in a notebook, using a sample covid-19 dataset. 
+- We'll forcast the relation between testing and active covid cases.
+
+**For a deeper dive**, try the [first-pipeline guide](https://docs.ploomber.io/en/latest/get-started/first-pipeline.html). 
```

Comment:
i think this should link to https://docs.ploomber.io/en/latest/get-started/basic-concepts.html

instead of the first-pipeline. we should delete the first-pipeline one

also since this one has a different name, we'll have to ensure there are no broken links here it in the ploomber repo

Reply:
I didn't think this is a replacement to the first-pipeline one, sounds like that's what you were thinking. I think we can discuss this since I don't think it's a good first pipeline example.